PendingReleaseNotes.md

```diff
@@ -11,3 +11,4 @@
 - Change `pspEnable` default value to `false` in helm charts.
 - [Bucket notifications and topics](https://rook.io/docs/rook/latest/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-notifications/)
   for object stores moved to stable from experimental.
+- Update `CephNodeNetworkPacketDrops` prometheus rule with relaxed alerting criteria
```

Comment:
These release notes are only for 1.11 feature. We can consider this as a fix to backport to 1.10, so I'd suggest reverting this file.